hardware/Netatmo.cpp

```diff
@@ -488,14 +526,146 @@ uint64_t CNetatmo::convert_mac(std::string mac)
 }
 
 
+uint64_t getValueFromString(const std::string& value)
+{
+	uint64_t result;
+	std::istringstream iss( value );
+	iss >> result;
+	return result;
+}
+
+std::string getStringFromValue(const uint64_t value)
+{
+	std::stringstream stream;
+	stream << value;
+	std::string string (stream.str());
+	return string;
+}
+
+/// <summery>
+/// Merge old and new devices, bu deleting duplicate log records and
+/// copy all other log records from the new to the old device before
+/// updating the old device to the new devices and finaly remove the new device.
+/// Note: the old device will become the new device.
+/// </summary>
+bool CNetatmo::MergeDevices (const uint64_t ipOldDeviceId, const uint64_t ipNewDeviceId)
```

Comment:
Please remove the space after the function name

Reply:
OK

Reply:
Done